daemon/policy.go

```diff
@@ -54,7 +54,13 @@ func (d *Daemon) TriggerPolicyUpdates(force bool) *sync.WaitGroup {
 //
 // Must be called with e.Consumable.Mutex and d.GetPolicyRepository().Mutex held.
 func (d *Daemon) EnableEndpointPolicyEnforcement(e *endpoint.Endpoint) (ingress bool, egress bool) {
-	// First check if policy enforcement should be enabled at the daemon level.
+	// If the endpoint has the reserved:init label, i.e. if it has not yet
+	// received any identity, always enforce policy (default deny).
```

Comment:
Same as my comment above:

This shouldn't be the case in my opinion. If policy enforcement for the daemon is set to `NeverEnforce`, then the policy corresponding to `reserved:init` should not apply at all. In other words, the policy applying to `reserved:init` should behave just like any other policy in conduit with the daemon's policy enforcement mode.

Reply:
Fixed.